src/nvim/ops.c

```diff
@@ -2311,6 +2311,13 @@ void op_insert(oparg_T *oap, long count1)
         }
       }
       bd.textcol = bd2.textcol;
+      /*
+       * If the insert was in the indent then include the indent
+       * change in the new text, otherwise don't.
+       */
+      if (did_indent && bd.textcol > ind_pre) {
+          bd.textcol += ind_post - ind_pre;
+      }
```

Comment:
```suggestion
      if (did_indent && bd.textcol > ind_pre) {
          // If the insert was in the indent then include the indent
          // change in the new text, otherwise don't.
          bd.textcol += ind_post - ind_pre;
      }
```

Reply:
Man, did I get the style guide wrong 🤦🏻 

(Sorry, @dundargoc !)